### PR TITLE
Fix docs issues

### PR DESCRIPTION
**Pull request by chenliang613**
Fix docs issues
* quick-start-guide
* configuration-parameters
* data-management

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/736/

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ This section provides the details of all the configurations required for CarbonD
...
 | high.cardinality.identify.enable | true | If the parameter is true, the high cardinality columns of the dictionary code are automatically recognized and these columns will not be used as global dictionary encoding. If the parameter is false, all dictionary encoding columns are used as dictionary encoding. The high cardinality column must meet the following requirements: value of cardinality > configured value of high.cardinalityEqually, the value of cardinality is higher than the threshold.value of cardinality/ row number x 100 > configured value of high.cardinality.row.count.percentageEqually, the ratio of the cardinality value to data row number is higher than the configured percentage. |
-| high.cardinality.threshold | 1000000 | Threshold to identify whether high cardinality column.Configuration value formula: Value of cardinality > configured value of high.cardinality. The minimum value is 10000. |
+| high.cardinality.threshold | 1000000 | Threshold to identify whether high cardinality column.if columns of cardinality > the configured value, then the columns don't do dictionary encoding. |
```
Comment (PallaviSingh1992) [inline docs/configuration-parameters.md]: replace 
| high.cardinality.threshold | 1000000 | It is a threshold to identify high cardinality of the column. If the value of column cardinality > the configured value, then the column is excluded from dictionary encoding. |

File: docs/configuration-parameters.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ This section provides the details of all the configurations required for CarbonD
  
 | Parameter | Default Value | Description |
 |----------------------------------------|--------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| spark.driver.memory | 1g | Amount of memory to use for the driver process, i.e. where SparkContext is initialized. NOTE: In client mode, this config must not be set through the SparkConf directly in your application, because the driver JVM has already started at that point. Instead, please set this through the --driver-memory command line option or in your default properties file. |
+| spark.driver.memory | 1g | Amount of memory to use for the driver process. |
```
Comment (PallaviSingh1992) [inline docs/configuration-parameters.md]: Amount of memory to be used by the driver process.

File: docs/configuration-parameters.md [hunk 3]
```diff
@@ -142,7 +142,7 @@ This section provides the details of all the configurations required for CarbonD
...
 | spark.executor.memory | 1g | Amount of memory to use per executor process. |
```
Comment (PallaviSingh1992) [inline docs/configuration-parameters.md]: We can replace the description by : Amount of memory to be used per executor process.

File: docs/configuration-parameters.md (hunk 2 above)
Comment (chenliang613) [inline docs/configuration-parameters.md]: accept.

File: docs/configuration-parameters.md (hunk 3 above)
Comment (chenliang613) [inline docs/configuration-parameters.md]: accept.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/737/

File: docs/quick-start-guide.md [hunk 4]
```diff
@@ -70,24 +70,22 @@ val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession(
 ##### Creating a Table
 
 ```
-scala>carbon.sql("create table if not exists test_table
-                (id string, name string, city string, age Int)
-                STORED BY 'carbondata'")
+scala>carbon.sql("create table if not exists test_table(id string, name string, city string, age Int)STORED BY 'carbondata'")
```
Comment (jackylk) [inline docs/quick-start-guide.md]: Use capital letter for SQL keywords

Comment (jackylk) [inline docs/quick-start-guide.md]: Use capital letter for SQL keywords

Comment (chenliang613) [inline docs/quick-start-guide.md]: OK

Comment (chenliang613) [inline docs/quick-start-guide.md]: OK

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/746/

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**